docs/conf.py

```diff
@@ -74,6 +74,9 @@
     "https://foo.org/",
     # GNU's site is a bit unreliable
     "https://www.gnu.org/.*",
+    # Github tag page for an upcoming release doesn't exist when we're writing
```

Comment:
Do you want to cherry-pick this instead so the merge to main is clean?

https://github.com/canonical/craft-parts/commit/23ad61303e9ad5efe208fa322da2a223185e725a